datasetinsights/datasets/transformers/base.py

```diff
@@ -0,0 +1,47 @@
+from abc import ABC, abstractmethod
+
+_registry = {}
```

Comment:
Can this registry be a class variable of `DatasetTransformer`?

Reply:
Yes, done.